examples/authn/authn_examples/user_actions.py

```diff
@@ -22,18 +23,41 @@ def main():
 
     try:
         print("Creating user...")
-        response = authn.user.create(
-            email=USER_EMAIL, authenticator=PASSWORD_INITIAL, id_provider=IDProvider.PASSWORD, profile=PROFILE_INITIAL
+        print("Start flow with signup and signin")
+        start_resp = authn.flow.start(
+            email=USER_EMAIL, flow_types=[m.FlowType.SIGNUP, m.FlowType.SIGNIN], cb_uri=CB_URI
         )
-        # Save user id for future use
-        user_id = response.result.id
-        print("User creation success. Result: ", response.result)
 
-        print("\n\nUser login...")
-        response = authn.user.login.password(email=USER_EMAIL, password=PASSWORD_INITIAL)
-        # Save user token to change password
-        user_token = response.result.active_token.token
-        print("User login success. Result: ", response.result)
+        print("Update flow with password")
```

Comment:
This code should work but it is assuming what choices will be available at each stage.  It would be a better example to follow if it looked at the response at each stage to see if the choice that it expected is present or not.  For example, after /v2/flow/start, it is assuming that there will be a "password" choice.  Then after submitting the password it is assuming that there will be a "profile" choice. etc..

Reply:
Yes, that's true